website/docs/r/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,92 @@
+* `flood_on_encap` - (Optional) Handling of L2 Multicast/Broadcast and Link-Layer traffic at EPG level. Control at EPG level if the traffic L2 Multicast/Broadcast and Link Local Layer should be flooded only on ENCAP or based on bridg-domain settings. Allowed values are "disabled", "enabled", and default value is "disabled". Type: String.
+* `label_match_criteria` - (Optional) Provider Label Match Criteria. Allowed values are "All", "AtleastOne", "AtmostOne", "None", and default value is "AtleastOne". Type: String.
+* `preferred_group_member` - (Optional) Represents parameter used to determine if the Cloud Service EPG is part of a group that does not a contract for communication. Allowed values are "exclude", "include", and default value is "exclude". Type: String.
+* `prio` - (Optional) The QoS priority class identifier. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+* `cloud_service_epg_type` - (Optional) The specific service type of the object or component. Allowed values are "Azure-ADDS", "Azure-AksCluster", "Azure-ApiManagement", "Azure-ContainerRegistry", "Azure-CosmosDB", "Azure-Databricks", "Azure-KeyVault", "Azure-Redis", "Azure-SqlServer", "Azure-Storage", "Azure-StorageBlob", "Azure-StorageFile", "Azure-StorageQueue", "Azure-StorageTable", "Custom", "Unknown", and default value is "Unknown". Type: String.
+* `relation_cloud_rs_cloud_epg_ctx` - (Optional) Represents the relation to a Relationship to the VRF of belonging (class fvCtx). Type: String.
+* `relation_fv_rs_cons` - (Optional) A block representing the relation to a Contract Consumer (class vzBrCP). The Consumer contract profile information. Type: Block.
+  * `prio` - (Optional) The system class determines the quality of service and priority for the consumer traffic. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+  * `target_dn` - (Required) The distinguished name of the target. Type: String
+* `relation_fv_rs_cons_if` - (Optional) A block representing the relation to a Contract Interface (class vzCPIf). A contract for which the EPG will be a consumer. Type: Block.
+  * `prio` - (Optional) The contract interface priority. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+  * `target_dn` - (Required) The distinguished name of the target. Type: String
+* `relation_fv_rs_cust_qos_pol` - (Optional) Represents the relation to a Custom QOS Policy (class qosCustomPol). A source relation to a custom QoS policy that enables different levels of service to be assigned to network traffic, including specifications for the Differentiated Services Code Point (DSCP) value(s) and the 802.1p Dot1p priority. This is an internal object. Type: String.
```

Comment:
-> Represents the relation to a Custom QOS Policy (class qosCustomPol) that enables different levels of service to be assigned to network traffic, including specifications for the Differentiated Services Code Point (DSCP) value(s) and the 802.1p priority. Type: String.